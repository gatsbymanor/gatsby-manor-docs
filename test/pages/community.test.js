import React from 'react';
import Community from '../../src/pages/community';

describe('<Community />', () => {
  const component = mount(
    <Community  />
  );

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
