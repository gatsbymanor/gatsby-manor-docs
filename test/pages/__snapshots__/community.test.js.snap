// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Community /> should render correctly 1`] = `
<Community>
  <div>
    <div
      className="community--banner"
    >
      <h2
        className="community--banner--title"
      >
        Community
      </h2>
    </div>
    <div
      className="container mr-auto community--section"
    >
      <div>
        <h2
          className=""
        >
          How To Get Support
        </h2>
        <p>
          There are many ways to get in contact with the Gatsby Manor         team. An engineer is often around and available for questions.
        </p>
      </div>
      <div>
        <h2
          className=""
        >
          Github
        </h2>
        <p>
          Any questions about the Jay, Themes, or Gatsby Manor can be directed to 
          <a
            href="https://github.com/gatsbymanor"
          >
            Github
          </a>
          . Find the repo you are interested in and create an issue detailing your problem. A core member will join the conversation when they are free. PR's are welcome for easy fixes or brand new features!
        </p>
      </div>
      <div>
        <h2
          className=""
        >
          Twitter
        </h2>
        <p>
          Need our attention? Tweet us 
          <a
            href="https://twitter.com/TheGatsbyManor"
          >
            @TheGatsbyManor
          </a>
           with your news. Tell us what cool things you are building. We love to publish the communities' creations on the site!
        </p>
      </div>
    </div>
  </div>
</Community>
`;
